torchaudio/transforms.py

```diff
@@ -780,6 +780,11 @@ class ComplexNorm(torch.nn.Module):
 
     Args:
         power (float, optional): Power of the norm. (Default: to ``1.0``)
+
+    Example
+        >>> complex_tensor = ... #  Tensor shape of (…, complex=2)
+        >>> transform = transforms.ComplexNorm(power=2)
+        >>> complex_tensor = transform(complex_tensor)
```

Comment:
```suggestion
        >>> complex_norm = transform(complex_tensor)
```

Reply:
No problems @nateanl. I have made the changes as per your suggestion :)